examples/3l-node/cli.rs

```diff
@@ -138,7 +138,7 @@ fn setup_editor(history_path: &Path) -> Editor<CommandHinter, DefaultHistory> {
     hints.insert(CommandHint::new("nodeinfo", "nodeinfo"));
     hints.insert(CommandHint::new("lspfee", "lspfee"));
     hints.insert(CommandHint::new(
-        "calculatelspfee <amount in millisat>",
+        "calculatelspfee <amount in sat>",
```

Comment:
```suggestion
        "calculatelspfee <amount in sats>",
```

Reply:
We've been very inconsistent with that. Myself I would prefer `SAT` in capital letters (for plural and singular). This is also how it's being displayed in the Lipa Wallet.